### PR TITLE
Added metadata to label.py and whisker.py

### PR DESCRIPTION
**Pull request by wenebob**
All pull requests must have an associated issue in the issue tracker. If there
isn't one, please go open an issue describing the defect, deficiency or desired
feature. You can read more about our issue and PR processes in the
[wiki](https://github.com/bokeh/bokeh/wiki/BEP-1:-Issues-and-PRs-management).

- [ ] issues: ref #11765
- [ ] tests added / passed
- [ ] release document entry (if new feature or API change)

### CODE REVIEW

File: examples/basic/annotations/whisker.py [hunk 1]
```diff
@@ -1,3 +1,14 @@
+''' A marker plot that shows the relationship between Car class and HWY from the autommpg sample data. 
+This example demonstrates the use of Whisker to display quintile ranges in the plot.
```
Comment (mosc9575) [inline examples/basic/annotations/whisker.py]:
```suggestion
This example demonstrates the use of Whisker to display quantile ranges in the plot.
```

This typo also exists in the title of the figure and should be corrected there as well.

Comment (wenebob) [reply examples/basic/annotations/whisker.py]: Thank you for this correction @mosc9575

File: examples/basic/annotations/label.py [hunk 2]
```diff
@@ -1,3 +1,12 @@
+''' A scatter plot that displays a label.
```
Comment (bryevdv) [inline examples/basic/annotations/label.py]: suggest:
```suggestion
''' A scatter plot that demonstrates different ways of adding labels.
```

File: examples/basic/annotations/whisker.py [hunk 3]
```diff
@@ -1,3 +1,14 @@
+''' A marker plot that shows the relationship between Car class and HWY from the autommpg sample data. 
+This example demonstrates the use of Whisker to display quantile ranges in the plot.
```
Comment (bryevdv) [inline examples/basic/annotations/whisker.py]: I think "class" here might be misunderstood as an OO "class"

```suggestion
''' A marker plot that shows the relationship between car type and highway MPG from the autompg
sample data.  This example demonstrates the use of whiskers to display quantile ranges in the plot.
```

**Review by bryevdv — CHANGES_REQUESTED** [review]
Thanks @Faith-Nchifor just a few comments!

**Comment by wenebob** [comment]
Thank you for these corrections @bryevdv 👍 
It's all good now.

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @Faith-Nchifor ! will merge when green

File: examples/basic/annotations/whisker.py [hunk 4]
```diff
@@ -1,3 +1,14 @@
+''' A marker plot that shows the relationship between car type and highway MPG from the autompg
+sample data.  This example demonstrates the use of whiskers to display quantile ranges in the plot.
```
Comment (mosc9575) [inline examples/basic/annotations/whisker.py]:
```suggestion
sample data. This example demonstrates the use of whiskers to display quantile ranges in the plot.
```

Here is an unnecessary whitespace.

Comment (bryevdv) [reply examples/basic/annotations/whisker.py]: Ahh good catch @mosc9575 ! @Faith-Nchifor please make this last edit, and *then* I will merge :D

Comment (wenebob) [reply examples/basic/annotations/whisker.py]: Thank you @mosc9575 I never even noticed!

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
Thanks @Faith-Nchifor !
